### DISCUSSION TITLE
解析只能解析1080p

### DISCUSSION THREAD

**Discussion by PetraLeyte** [yaobiao131/downkyicore#73] [General]
4月12凌晨解析个视频只能解析1080p，重新登录过（有大会员）升级到最新的1.0.9版本了，但是只能解析1080p。昨天还可以解析4k

**Comment by yaobiao131** [comment]
麻烦提供视频链接

**Reply by PetraLeyte** [reply]
现在又恢复了、可以解析4k了。可能凌晨时候出现了bug？
